### DISCUSSION TITLE
Create new screenshot

### DISCUSSION THREAD

**Discussion by RustoMCSpit** [linuxmint/discussions#279] [Ideas]
So under Create New Document there should be 'Create New Screenshot' under 'Empty Document'

**Comment by ElmNumaeStNo37** [comment]
> 'Create New Screenshot'

it would be helpful if you can elaborate what you mean with this

**Comment by 100savage** [comment]
Open screenshot with a simple right-click on the desktop.

Create a file called **screenshot.nemo_action** and place it in your **.local/share/nemo/actions/**   directory
Note: the .local folder is hidden and located in your home directory.
Put the below text in that file

```
[Nemo Action]
Name=ScreenShot
Comment=Save images of your screen or individual windows
Exec=gnome-screenshot --interactive
Icon-Name=org.gnome.Screenshot
Selection=None
Extensions=any;
Conditions=desktop;
```
Open a terminal and type the below command to restart nemo
`nemo -q`

Right-click on your desktop and select screenshot

**Reply by goyalyashpal** [reply]
just jor records, this is shared under tutorials as new discussion:
* https://github.com/orgs/linuxmint/discussions/284
